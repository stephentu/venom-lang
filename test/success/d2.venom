x = { 'a' : 10, 'b' : 20 };
print(x.get('a'));
print(x.get('b'));

y = { 10 : 100, 100 : 1000 };
print(y.get(10));
print(y.get(100));
