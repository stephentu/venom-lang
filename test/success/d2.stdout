10
20
100
1000
